### PR TITLE
Document deployment process

### PR DESCRIPTION
**Pull request by tsusanka**
Documents deployment process

### CODE REVIEW

File: docs/deployment/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+1. Go to develop branch.
+1. Make sure that CHANGELOG is up to date, usually it is not so walk thru commits and update it.
+1. Run `make version-patch` - this will update version number in multiple files.
+1. Commit these changes with the Changelog changes into `develop` with message "version X.X.X" ([example commit](https://github.com/trezor/connect/commit/d28e506524bc666757715294f7d030ea1a1d6eef)).
+1. Prepare PR from `develop` to `v8` branch, paste changelog as description and tag @prusnak or @tsusanka ([example PR](https://github.com/trezor/connect/pull/812)).
+1. Review and merge.
+
+
+## Deploying
+1. After the merge we deploy Connect to connect.trezor.io. TODO description by @prusnak.
```
Comment (tsusanka) [inline docs/deployment/index.md]: @prusnak could you please describe what steps need to be done to deploy Connect? Just dumping it in comment is fine, I'll add it.

Comment (prusnak) [reply docs/deployment/index.md]: to deploy version 8:

```
git checkout v8
make clean
make build
make sync-8
```

File: docs/deployment/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+1. `cd ../` - go back
+1. `yarn build:npm-extended` - build extended library (full for us)
+1. `cd ./npm-extended`
+1. `npm publish --tag extended` - publish extended build
+
+[1] Make sure that you do have npm account and access to write to trezor-connect package.
+
+### Beta
+
+If you want to publish to npm as `beta` (or any other not official tag) do the following
```
Comment (szymonlesisz) [inline docs/deployment/index.md]:
```suggestion
If you want to publish to npm as `beta` (from any branch, production release is not required) do the following:
`

File: docs/deployment/index.md (hunk 1 above)
Comment (tsusanka) [reply docs/deployment/index.md]: Thanks. Added.

**Comment by tsusanka** [comment]
So this should be done! Feel free to check and merge @szymonlesisz.

**Pull request merged**